execution/rpcserver/server.go

```diff
@@ -0,0 +1,156 @@
+package executionrpcserver
+
+import (
+	"context"
+
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/execution"
+)
+
+type ExecutionRPCServer struct {
+	executionClient execution.ExecutionClient
+	// executionSequencer   execution.ExecutionSequencer
+	// executionRecorder    execution.ExecutionRecorder
+	// executionBatchPoster execution.ExecutionBatchPoster
+}
+
+func NewExecutionRpcServer(executionClient execution.ExecutionClient) *ExecutionRPCServer {
```

Comment:
nitpick: golang naming convention

```suggestion
func NewExecutionRPCServer(executionClient execution.ExecutionClient) *ExecutionRPCServer {
```